modules/Core/queries/admin_users.php

```diff
@@ -73,19 +75,27 @@
         $obj->joined = date('d M Y', $result->joined);
 
         // Get group
-        $group = DB::getInstance()->selectQuery('SELECT `name` FROM nl2_groups g JOIN nl2_users_groups ug ON g.id = ug.group_id WHERE ug.user_id = ? ORDER BY g.order ASC LIMIT 1', [$result->id]);
+        $group = DB::getInstance()->query('SELECT `name` FROM nl2_groups g JOIN nl2_users_groups ug ON g.id = ug.group_id WHERE ug.user_id = ? ORDER BY g.order ASC LIMIT 1', array($result->id));
         $obj->groupName = $group->first()->name;
 
+        $obj->quick_action = array('
```

Comment:
Looks like the template files haven't been uploaded in this PR to show the quick actions :(
Any idea what happened to them?